terra_accounts/managers.py

```diff
@@ -1,6 +1,7 @@
 from django.contrib.auth.base_user import BaseUserManager
 from django.contrib.contenttypes.models import ContentType
 from django.db.models import Manager, ObjectDoesNotExist
+from terra_settings.settings import TERRA_APPLIANCE_SETTINGS
```

Comment:
The recommanded way to import settings in django app is:

```suggestion
from django.conf import settings
```